content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+
+For testing, you can mint a token directly from the Candy Machine v2 CLI using the `mint_one_token` command.
+
+### Mint using the Candy Machine UI
+>>>>>PUT SOME STUFF HERE
+
+### Mint using Captcha
+>>>>>PUT SOME STUFF HERE
+
+### Allow whitelisted mints
+>>>>>PUT SOME STUFF HERE
+
+## Withdraw candy machine rent
+
+Once a candy machine is fully minted, the data stored in the candy machine account is no longer relevant. At that point, the account can be close and the rent for the account recovered using the `withdraw` command. The `withdraw` command must be executed with the keypair that created the Candy Machine.
+
+
+# Demo
+
+Let’s put all of this into practice by creating a Candy Machine and minting our collection using the Candy Machine UI.
+
+### 1. Download the starter code
+
+Let's begin by downloading the starter [code](https://github.com/Unboxed-Software/solana-metaplex-intro). The starter code includes an `assets` folder and the configuration file for our Candy Machine, `config.json`. Additionally, it includes two helper scripts. One creates a new keypair we will use for this lesson. The other creates the whitelist token we will later use to enable whitelist settings.
+
+### 2. Setup
+While in the project's root directory, lets first install the dependencies for our starter code:
+```sh
+npm install
+```
+
+Next, let's set up a new keypair to use for this lesson and airdrop some devnet SOL using our helper script. Run the following command:
+
+```sh
+npm start
+```
+You should now see a new `private-key.json` file. Copy the keypair from the new file and import it into Phantom.
+
+![Gif of Phantom Import](../assets/solana-nft-phantom.gif)
+
+### 3. Download Metaplex
+
+Now that our project has a new keypair, we're ready to set up the Metaplex helpers. While in the project's root directory, clone the Metaplex repository:
+
+```sh
+git clone https://github.com/metaplex-foundation/metaplex.git
+```
+
+This repository contains the Candy Machine v2 CLI tools. If you want to have a look at them more closely, you can find them in `metaplex/js/packages/cli/src/candy-machine-cli-v2.ts`.
+
+We'll be using `ts-nod` to run commands. If you don't have `ts-node` installed globally, install it now:
+
+```sh
+npm install -g typescript
+npm install -g ts-node
+```
+
+Next, install the Metaplex package dependencies:
+
+```sh
+yarn install --cwd metaplex/js/
+```
+
+Finally, let's confirm that everything is working by checking the Candy Machine v2 version:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts --version
+```
+
+If this command logs a version to the console, you should be good to go. If not, you'll have to debug the problem(s) before moving on. Check the logs if you're having issues.
```

Comment:
@jamesrp13 sugg:

... logs the version to the console, ...

... logs a version number to the console, ...